tests/test_split_batch.py

```diff
@@ -76,6 +80,7 @@ def generate_maskrcnn_sample(num_detections,
 def dummy_batches(batch_size=12):
```

Comment:
If i'm reading the tests right, this only tests that it doesn't error. can you also add a test for getting the right shape out?